docs/swarm/swarm-tutorial/add-nodes.md

```diff
@@ -0,0 +1,69 @@
+<!--[metadata]>
++++
+title = "Add nodes to the Swarm"
+description = "Add nodes to the Swarm"
+keywords = ["tutorial, cluster management, swarm"]
+[menu.main]
+identifier="add-nodes"
+parent="swarm-tutorial"
+weight=13
++++
+<![end-metadata]-->
+
+## Add nodes to the Swarm
+
+Once you've [created a Swarm](create-swarm.md) with a manager node, you can add
+worker nodes.
+
+1. Open a terminal and ssh into the machine where you want to run a worker node.
+This tutorial uses the name `worker1`.
+
+2. Run the `docker swarm join` command to create a worker node joined to the
+existing Swarm. Pass the IP address of the manager node and the port where the
+manager listens:
+
+    ```
+    $ docker swarm join MANAGER-IP:PORT
+    ```
+
+    For example, to join the Swarm on `manager1` node for the tutorial:
+
+    ```
+    $ docker swarm join 192.168.99.100:2377
+
+    This node is attempting to join a Swarm.
+    ```
+
+3. Open a terminal and ssh into the machine where the manager node runs and run
+the `docker node ls` command to see the new worker node:
+
+    ```bashtext
```

Comment:
@sanscontext , will render properly with updated skin. cue is to prevent it from mangling the font color (which might be less of an issue now the column title doesn't have an apostrophe.